src/templates/globals.mustache.js

```diff
@@ -56,16 +59,23 @@ window.QUnitGlobals = {
                 start();
             }, WAIT_FOR_IFRAME_TIMEOUT);
 
-            intervalId = window.setInterval(function () {
-                if (iframe && iframe.contentWindow) {
-                    window.clearInterval(intervalId);
+            try {
+                isIframeLoaded = iframe.contentWindow && iframe.contentWindow.document.readyState === 'complete';
```

Comment:
I'm not sure but I think in some browsers `iframe.contentWindow.document` is null when the iframe is just added. It this case we go to the `catch` block and set the 'isIframeLoaded' to true but it should be false.
Lets check `iframe.contentWindow && iframe.contentWindow.document && ... === 'complete'`